### PR TITLE
WIP Fabo/donations

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI
image

### CODE REVIEW

File: src/components/common/TmBalance.vue [hunk 1]
```diff
@@ -119,7 +119,7 @@
         </div>
 
         <div class="table-cell rewards">
-          <h2 v-if="totalRewards > 0.001">
+          <h2>
```
Comment (faboweb) [inline src/components/common/TmBalance.vue:122]: revert

File: src/components/common/TmBalance.vue [hunk 2]
```diff
@@ -291,9 +291,7 @@ export default {
     // only be ready to withdraw of the validator rewards are loaded and the user has rewards to withdraw
     // the validator rewards are needed to filter the top 5 validators to withdraw from
     readyToWithdraw() {
-      return Object.values(this.totalRewardsPerDenom).find(
-        value => value > 0.001
-      )
+      return true
```
Comment (faboweb) [inline src/components/common/TmBalance.vue:294]: revert

**Comment by Bitcoinera** [comment]
Ah we really need to look into this linting issue. Makes readability of real changes difficult

File: src/components/common/TmBalance.vue (hunk 1 above)
Comment (faboweb) [reply src/components/common/TmBalance.vue:122]:
```suggestion
          <h2 v-if="totalRewards > 0.001">
```

File: src/components/common/TmBalance.vue (hunk 2 above)
Comment (faboweb) [inline src/components/common/TmBalance.vue:294]:
```suggestion
      return Object.values(this.totalRewardsPerDenom).find(
        value => value > 0.001	
      )
```

**Comment by Bitcoinera** [comment]
I guess we can safely close this one for now?

Legality entered once more in the way of creativity and awesomeness

**Comment by Bitcoinera** [comment]
Closing since we are not going to do this in the end 😞

**Pull request closed without merging**